sources/_subprojects/dex/_sources.yml

```diff
@@ -67,4 +67,7 @@ sources:
       - name: trades
   - name: cow_protocol_arbitrum
     tables:
-      - name: trades
\ No newline at end of file
+      - name: trades
+  - name: labels
+    tables:
+      - name: jelly_swap_pools_sei
```

Comment:
i know this is likely confusing, and we will look to document along with how to use subprojects in general, but we should list spells that are sources cross-project to be placed in directory path of where the spell is built. we will be moving `labels` spells to `daily_spellbook`, so you could proactively add to that file?